### PR TITLE
sanity price check on glarks

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: src/fights.ts [hunk 1]
```diff
@@ -886,9 +887,11 @@ const freeFightSources = [
   ),
 
   new FreeFight(
-    () => (get("questL11Ron") === "finished" ? 5 - get("_glarkCableUses") : 0),
+    () =>
+      get("questL11Ron") === "finished"
+        ? 5 - get("_glarkCableUses")
+        : 0 && have($item`glark cable`),
```
Comment (burningbman) [inline src/fights.ts:893]: Doesn't this always come out to 0?

**Review by burningbman — APPROVED** [review]

File: src/fights.ts (hunk 1 above)
Comment (horrible-little-slime) [reply src/fights.ts:893]: oh whoops

Comment (horrible-little-slime) [reply src/fights.ts]: How does it look now

**Review by burningbman — APPROVED** [review]

File: src/fights.ts (hunk 1 above)
Comment (burningbman) [reply src/fights.ts]: Looks good. Approved.

**Pull request merged**